src/community/pom.xml

```diff
@@ -28,83 +28,21 @@
-            <descriptor>release/ext-backup-restore.xml</descriptor>
-            <descriptor>release/ext-notification.xml</descriptor>
-            <descriptor>release/ext-jdbc-metrics.xml</descriptor>
-            <descriptor>release/ext-opensearch-eo.xml</descriptor>
-            <descriptor>release/ext-taskmanager-core.xml</descriptor>
-            <descriptor>release/ext-taskmanager-s3.xml</descriptor>
-            <descriptor>release/ext-netcdf-ghrsst.xml</descriptor>
-            <descriptor>release/ext-cog.xml</descriptor>
-            <descriptor>release/ext-s3-geotiff.xml</descriptor>
-            <descriptor>release/ext-ogr-datastore.xml</descriptor>
-            <descriptor>release/ext-jwt-headers.xml</descriptor>
-            <descriptor>release/ext-ogcapi-coverages.xml</descriptor>
-            <descriptor>release/ext-ogcapi-dggs.xml</descriptor>
-            <descriptor>release/ext-ogcapi-features.xml</descriptor>
-            <descriptor>release/ext-ogcapi-images.xml</descriptor>
-            <descriptor>release/ext-ogcapi-maps.xml</descriptor>
-            <descriptor>release/ext-ogcapi-styles.xml</descriptor>
-            <descriptor>release/ext-ogcapi-tiled-features.xml</descriptor>
-            <descriptor>release/ext-ogcapi-tiles.xml</descriptor>
-            <descriptor>release/ext-importer-jdbc.xml</descriptor>
-            <descriptor>release/ext-features-autopopulate.xml</descriptor>
-            <descriptor>release/ext-features-templating.xml</descriptor>
-            <descriptor>release/ext-gsr.xml</descriptor>
-            <descriptor>release/ext-smart-data-loader.xml</descriptor>
-            <descriptor>release/ext-cov-json.xml</descriptor>
-            <descriptor>release/ext-mongodb-schemaless.xml</descriptor>
-            <descriptor>release/ext-web-service-auth.xml</descriptor>
-            <descriptor>release/ext-gwc-mbtiles.xml</descriptor>
-            <descriptor>release/ext-vsi.xml</descriptor>
-            <descriptor>release/ext-libdeflate.xml</descriptor>
-            <descriptor>release/ext-stac-datastore.xml</descriptor>
-            <descriptor>release/ext-vector-mosaic-datastore.xml</descriptor>
-            <descriptor>release/ext-proxy-base-ext.xml</descriptor>
-            <descriptor>release/ext-imagemap.xml</descriptor>
-            <descriptor>release/ext-xslt.xml</descriptor>
-            <descriptor>release/ext-datadir-catalog-loader.xml</descriptor>
-            <descriptor>release/ext-wps-longitudinal-profile.xml</descriptor>
-            <descriptor>release/ext-monitor-kafka.xml</descriptor>
-            <descriptor>release/ext-graticule.xml</descriptor>
-            <descriptor>release/ext-wfs-freemarker.xml</descriptor>
-          </descriptors>
-          <finalName>geoserver-${project.version}</finalName>
-        </configuration>
+        <groupId>org.commonjava.maven.plugins</groupId>
+        <artifactId>directory-maven-plugin</artifactId>
+        <version>0.3.1</version>
```

Comment:
Move version to root pom plugin management section.

aside: It looks like version 1.0 is available as of Sep 22, 2021 which prevents parallel build warning.

Reply:
copied over from main... but yes, testing centralizing in dependency management and upgrading